src/env.h

```diff
@@ -459,8 +459,8 @@ class TickInfo : public MemoryRetainer {
   AliasedUint8Array fields_;
 };
 
-class TrackingTraceStateObserver :
-    public v8::TracingController::TraceStateObserver {
+class TrackingTraceStateObserver
+    : public v8::TracingController::TraceStateObserver {
```

Comment:
nit: this change looks unrelated, was it done by clang-format?

Reply:
Yes, I got a clang-format error and thought it couldn't be fixed without making a squashed commit, but it's not. Apologized for the confusion. Fixed.